frontend/src/views/CellGuide/components/common/OntologyDagView/index.tsx

```diff
@@ -74,7 +75,14 @@ interface TissueIdProps extends BaseTreeProps {
   tissueName: string;
 }
 
-type TreeProps = CellTypeIdProps | TissueIdProps;
+// Both cellTypeId and tissueId (with tissueName) are mandatory.
```

Comment:
this is set-up for the feature we will add where the cell ontology tree in a CellGuideCard will change based on the global tissue filter that is selected (#5368 )